yarn-error.log

```diff
@@ -0,0 +1,9071 @@
+Arguments: 
```

Comment:
This file is not needed, remove it.

Reply:
I thought I could grab a berth on the contributors ranking with this +9000 file, too bad I got called out!